tests/std/tests/GH_001914_Cached_position/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
I think that test name shouldn't start with uppercase letter (note that in `test.lst` it doesn't)

Reply:
You're right - and you managed to submit your review 4 minutes before me :joy_cat: I'll fix this.